Source: FC14
Status: committed upstream in r273, can be dropped past 1.4.9
--- a/src/scim_utility.cpp
+++ b/src/scim_utility.cpp
@@ -776,6 +776,8 @@
     { "pt_PT",    NULL, N_("Portuguese"), "Português", "@euro" },
     { "ro_RO",    NULL, N_("Romanian"), "Română", NULL },
     { "ru_RU",    NULL, N_("Russian"), "русский", ".koi8r" },
+    { "sd",    "sd_IN", N_("Sindhi"), "ﺲﻧڌﻱ", NULL },
+    { "sd_IN",    NULL, N_("Sindhi"), "सिन्धी", "@devanagari" },
     { "si_LK",    NULL, N_("Sinhala"), "සිංහල", NULL },
     { "sk_SK",    NULL, N_("Slovak"), "Slovenský", NULL },
     { "sl_SI",    NULL, N_("Slovenian"), "Slovenščina", NULL },
